roofit/histfactory/CMakeLists.txt

```diff
@@ -78,6 +81,14 @@ ROOT_STANDARD_LIBRARY_PACKAGE(HistFactory
     XMLParser
 )
 
+if(yml)
+  FIND_PACKAGE(ryml)
```

Comment:
This should go to https://github.com/root-project/root/blob/master/cmake/modules/SearchInstalledSoftware.cmake#L255
Please take an example from xxhash as well as to add support of option fail-on-missing and possibly builtins in the future.